src/components/AfterInitialMount/AfterInitialMount.tsx

```diff
@@ -0,0 +1,18 @@
+import React, {ReactNode} from 'react';
+import {useIsAfterInitialMount} from '../../utilities/use-is-after-initial-mount';
+
+interface Props {
+  children?: ReactNode;
+  fallback?: ReactNode;
+}
+
+function AfterInitialMount({children, fallback}: Props) {
+  const isMounted = useIsAfterInitialMount();
+  const content = isMounted ? children : fallback;
+
+  return <React.Fragment>{content}</React.Fragment>;
+}
+
+AfterInitialMount.defaultProps = {fallback: null};
```

Comment:
Adding default props to make this more testable, since we can't replicate a node env right now we won't be able to assert that null is rendered as a default for fallback however using default props we can assert that the default fallback is null.

Reply:
https://github.com/reactjs/rfcs/pull/107 suggests that defaultProps on functional components will be deprecated at some future point.

I'd prefer we do `function AfterInitialMount({children, fallback= null}: Props) {`

Reply:
Chat outcome: doing the js default makes it harder to test but as this is such a trivial thing I don't think much is lost by having it be untestable. Lets use the js default and remove this (and then have the export on the same line as the function declaration)